src/stores/AccountController.ts

```diff
@@ -470,3 +487,15 @@ async function afterLogin(
     getPublisherConsents(),
   ]);
 }
+
+const validateInputLength = (values: { firstName: string; lastName: string }) => {
```

Comment:
Should the 50 character limit be a property on the services like canSupportEmptyFullName?